.github/workflows/release.yml

```
@@ -0,0 +1,33 @@
+jobs:
+    clojars:
+        runs-on: ubuntu-latest
+        environment: env
+        defaults:
+            run:
+                shell: bash
+        steps:
+            - uses: actions/checkout@v4
+            - uses: "DeLaGuardo/setup-clojure@12.5"
+              with:
+                cli: "1.11.1.1139"
```

yamllint

[error] wrong indentation: expected 18 but found 16

(indentation)